frame/instrumental-strategy-pablo/src/mock/helpers.rs

```diff
@@ -0,0 +1,67 @@
+use composable_traits::{defi::CurrencyPair, dex::Amm};
+use frame_support::{assert_ok, traits::fungibles::Mutate};
+use pallet_pablo::PoolInitConfiguration;
+use primitives::currency::CurrencyId;
+use sp_runtime::Permill;
+
+use crate::mock::{
+	account_id::{AccountId, ALICE, BOB},
+	runtime::{Balance, BlockNumber, Pablo, PoolId, Tokens},
+};
+
+// TODO(saruman9): will be used in the future
+#[allow(dead_code)]
+pub fn create_usdt_usdc_pool() -> PoolId {
+	let usdt_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let usdc_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let assets = CurrencyPair::new(CurrencyId::USDT, CurrencyId::USDC);
+	let amounts = vec![usdt_amount, usdc_amount];
+	create_pool(assets, amounts, Permill::zero(), Permill::from_percent(50))
+}
+
+// TODO(saruman9): doesn't work, because PICA is a native token in the runtime
+#[allow(dead_code)]
+pub fn create_pica_ksm_pool() -> PoolId {
+	let pica_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let ksm_amount = 1_000_000_000 * CurrencyId::unit::<Balance>();
+	let assets = CurrencyPair::new(CurrencyId::PICA, CurrencyId::KSM);
+	let amounts = vec![pica_amount, ksm_amount];
+	create_pool(assets, amounts, Permill::zero(), Permill::from_percent(50))
+}
+
+pub fn create_layr_crowd_loan_pool() -> PoolId {
```

Comment:
these three create_x_y_pools can be combined into a function `create_pool_with(asset_1, asset_2)`

Reply:
you can also add a parameter to allow specifying different balances of each token

Reply:
Stupid oversight, I was in a hurry to fix it. Thanks.

Fixed by d714511.